.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Version
+[provide output of the `$ openshift version` command]
+
+## Description
+
+## How reproducible
+
+1. [step 1]
+2. [step 2]
+
+## Expected result
+
+## Current result
+
+## More information
+
+[try to run `$ oadm diagnostics` command if possible]
+[visit https://docs.openshift.org/latest/welcome/index.html]
```

Comment:
@bparees oc get all -o json > i'm scared people will be pasting this into the issues :)